contracts/src/BeefyClient.sol

```diff
@@ -428,6 +443,27 @@ contract BeefyClient {
         return Bitfield.subsample(ticket.prevRandao, bitfield, ticket.numRequiredSignatures, ticket.validatorSetLen);
     }
 
+    function setInitialCheckpoint(
+        uint64 _initialBeefyBlock,
+        ValidatorSet calldata _initialValidatorSet,
+        ValidatorSet calldata _nextValidatorSet
+    ) external {
+        if (msg.sender != bootstrapOperator) {
+            revert Unauthorized();
+        }
+        delete bootstrapOperator;
```

Comment:
Instead of making this function single-use by deleting `bootstrapOperator`, we could have a separate function to renounce ownership. This will be useful in rococo testnet as we will never renonce ownership making recovery from beefy issues easier.

Reply:
Agree. Though a bit controversial leaving that owner control will make recovery easier, especially in Beefy's early phase applied in production.